bin/millau/node/src/service.rs

```diff
@@ -207,6 +207,7 @@ pub fn new_full(mut config: Configuration) -> Result<TaskManager, ServiceError>
 	}
```

Comment:
There are short instructions on top of `service.rs` of both Millau and Rialto nodes. Could you, please, update these to (at least) mention that beefy integration should be re-added/preserved on update. These file are a major pain when we update Substrate/Polkadot refs => it is often easier to replace the whole file with upstream version and then fix issues/re-add something that we have customized.